docs/reference/slm/getting-started-slm.asciidoc

```diff
@@ -62,66 +77,56 @@ PUT /_slm/policy/nightly-snapshots
-    <<schedule-cron,Cron syntax>>, in this 
-    case at 1:30AM each day
-<2> whe name each snapshot should be given, using 
-    <<date-math-index-names,date math>> to include the current date in the name
-    of the snapshot
-<3> the repository the snapshot should be stored in
-<4> the configuration to be used for the snapshot requests (see below)
-<5> which indices should be included in the snapshot, in this case, every index
-<6> Optional retention configuration
-<7> Keep snapshots for 30 days
-<8> Always keep at least 5 successful snapshots
-<9> Keep no more than 50 successful snapshots, even if they're less than 30 days old
-
-This policy will take a snapshot of every index each day at 1:30AM UTC.
-Snapshots are incremental, allowing frequent snapshots to be stored efficiently,
-so don't be afraid to configure a policy to take frequent snapshots.
-
-In addition to specifying the indices that should be included in the snapshot,
-the `config` field can be used to customize other aspects of the snapshot. You
-can use any option allowed in <<snapshots-take-snapshot,a regular snapshot 
-request>>, so you can specify, for example, whether the snapshot should fail in
-special cases, such as if one of the specified indices cannot be found.
+<1> When the snapshot should be taken in
+    <<schedule-cron,Cron syntax>>: daily at 1:30AM UTC
+<2> How to name the snapshot: use  
+    <<date-math-index-names,date math>> to include the current date in the snapshot name
+<3> Where to store the snapshot
+<4> The configuration to be used for the snapshot requests (see below)
+<5> Which indices to include in the snapshot: all indices
+<6> Optional retention policy: keep snapshots for 30 days, 
+retaining at least 5 and no more than 50 snapshots regardless of age. 
+
+You can specify additional snapshot configuration options to customize how snapshots are taken.
+For example, you could configure the policy to fail the snapshot 
+if one of the specified indices is missing. 
+For more information about snapshot options, see <<snapshots-take-snapshot,snapshot requests>>.
 
 [float]
 [[slm-gs-test-policy]]
 ==== Test the snapshot policy
 
-While snapshots taken by SLM policies can be viewed through the standard snapshot
-API, SLM also keeps track of policy successes and failures in ways that are a bit
-easier to use to make sure the policy is working.  Once a policy has executed at
-least once, when you view the policy using the <<slm-api-get-policy>>, 
-some metadata will be returned indicating whether the snapshot was sucessfully 
-initiated or not.
+A snapshot taken by {slm-init} is just like any other snapshot. 
+You can view information about snapshots in {kib} or get info with the snapshot APIs. 
+In addition, {slm-init } keeps track of policy successes and failures so you 
+have insight into how the policy is working. If the policy has executed at
+least once, the <<slm-api-get-policy>> API returns additional metadata
```

Comment:
I typically define anchor text. You may want to see how this renders to ensure its not italicized.
> <<slm-api-get-policy>> API